website/content/api-docs/json-jobs.mdx

```diff
@@ -8,146 +8,234 @@ description: |-
-          "Attempts": 10,
-          "Delay": 30000000000,
-          "DelayFunction": "exponential",
-          "Interval": 0,
-          "MaxDelay": 3600000000000,
-          "Unlimited": true
-        },
+        "ReschedulePolicy": null,
         "EphemeralDisk": {
+          "Sticky": null,
+          "Migrate": null,
           "SizeMB": 300
-        }
+        },
+        "Update": null,
+        "Migrate": null,
+        "Networks": [
+          {
+            "Mode": "",
+            "Device": "",
+            "CIDR": "",
+            "IP": "",
+            "DNS": null,
+            "ReservedPorts": null,
+            "DynamicPorts": [
+              {
+                "Label": "db",
+                "Value": 0,
+                "To": 6379,
+                "HostNetwork": ""
+              }
+            ],
+            "Hostname": "",
+            "MBits": null
+          }
+        ],
+        "Meta": null,
+        "Services": [
+          {
+            "Id": "",
+            "Name": "redis-cache",
+            "Tags": [
+              "global",
+              "cache"
+            ],
+            "CanaryTags": null,
+            "EnableTagOverride": false,
+            "PortLabel": "db",
+            "AddressMode": "",
+            "Checks": null,
+            "CheckRestart": null,
+            "Connect": null,
+            "Meta": null,
+            "CanaryMeta": null,
+            "TaskName": "",
+            "OnUpdate": "",
+            "Provider": ""
+          }
+        ],
+        "ShutdownDelay": null,
+        "StopAfterClientDisconnect": null,
+        "MaxClientDisconnect": null,
+        "Scaling": null,
+        "Consul": null
       }
     ],
     "Update": {
+      "Stagger": null,
       "MaxParallel": 1,
+      "HealthCheck": null,
       "MinHealthyTime": 10000000000,
       "HealthyDeadline": 180000000000,
+      "ProgressDeadline": 600000000000,
+      "Canary": 0,
       "AutoRevert": false,
-      "Canary": 0
-    }
+      "AutoPromote": null
+    },
+    "Multiregion": null,
+    "Spreads": null,
+    "Periodic": null,
+    "ParameterizedJob": null,
+    "Reschedule": null,
+    "Migrate": {
+      "MaxParallel": 1,
+      "HealthCheck": "checks",
+      "MinHealthyTime": 10000000000,
+      "HealthyDeadline": 300000000000
+    },
+    "Meta": null,
+    "ConsulToken": null,
+    "VaultToken": null,
+    "Stop": null,
+    "ParentID": null,
+    "Dispatched": false,
+    "DispatchIdempotencyToken": null,
+    "Payload": null,
+    "ConsulNamespace": null,
+    "VaultNamespace": null,
+    "NomadTokenID": null,
+    "Status": null,
+    "StatusDescription": null,
+    "Stable": null,
+    "Version": null,
+    "SubmitTime": null,
+    "CreateIndex": null,
+    "ModifyIndex": null,
+    "JobModifyIndex": null
   }
 }
 ```
 
-The example JSON could be submitted as a job using the following:
+Prior to Nomad 1.3 the example JSON could be submitted as a job using the
+third party curl command:
```

Comment:
You still can in Nomad 1.3 as well. Is this referring to `nomad operator api` and if so, should we call that out here?

Reply:
Yeah, not sure what I was thinking. I'm going to lean into our fancy new versioned docs by removing this. I already cover how to run JSON jobs up top.